app/views/contexts/submit_context.py

```diff
@@ -25,21 +29,30 @@ def __call__(self, collapsible=True, answers_are_editable=True):
         warning = submission_schema.get("warning") or None
 
         context = {
-            "summary": {
-                "groups": groups,
-                "answers_are_editable": answers_are_editable,
-                "collapsible": collapsible,
-                "summary_type": "Summary",
-            },
             "title": title,
             "guidance": guidance,
             "warning": warning,
             "submit_button": submit_button,
         }
+        if include_summary:
+            context.update(self._get_summary_context(collapsible, answers_are_editable))
```

Comment:
If `_get_summary_context` returned a dict without the `summary` key, this could be written as:
```suggestion
            context["summary"] = self._get_summary_context(collapsible, answers_are_editable)
```

Reply:
Done